cpp/cmake/thirdparty/get_thrust.cmake

```diff
@@ -17,11 +17,15 @@
 function(find_and_configure_thrust)
   include(${rapids-cmake-dir}/cpm/thrust.cmake)
 
+
```

Comment:
These changes to `CPM_DOWNLOAD_ALL` aren't needed since you have set `always_download` in the override_cpm.json